cli/packages/prisma-cli-core/src/commands/deploy/deploy.ts

```diff
@@ -320,11 +320,11 @@ ${chalk.gray(
       const isPackaged = fs.existsSync('/snapshot')
       const spawnPath = isPackaged ? nativeSpawnSync : spawnSync
       const child = spawnPath(splittedHook[0], splittedHook.slice(1))
-      const stderr = child.stderr && child.stderr.toString()
+      const stderr = child.stderr && child.stderr.toString().trim()
```

Comment:
What's the purpose of the `.trim()`?

Reply:
There were two empty spaces for one `stderr`, making the condition below this pass. Trim removes trailing spaces making this more reliable. 

Is there any downside of using `trim` that you can think of?

Reply:
It could just have unwanted side effects as this is affecting every command.
Let's merge it for now and observe in beta if anything unwanted happens.